### DISCUSSION TITLE
What does no-streaming in logs means?

### DISCUSSION THREAD

**Discussion by devrus91** [dotnet/yarp#2193] [Q&A] [answered]
I have some api which returns a file.
While requests are running I see logs like this:
` Proxying to http://content-api:8041/file/1 HTTP/2 RequestVersionOrLower no-streaming`

Does `no-streaming` means that before responding YARP read all request and than responds or this mewssage realtes to GRPC streaming?

**Comment by Tratcher** [comment]
Filed as an issue: https://github.com/microsoft/reverse-proxy/issues/2194

**Reply by Tratcher** [reply, marked as answer]
This is specific to gRPC streaming mode, causing YARP to proxy the request and response body simultaneously. Otherwise the response is not proxied until the request body finishes.
